SportsHub/SportsHub.Api/Program.cs

```diff
@@ -16,6 +19,26 @@
     app.UseSwaggerUI();
 }
 
+app.UseExceptionHandler(error =>
+{
```

Comment:
Please create separate class for exception handling.

Reply:
Okey . Soon as possible it will be implemented

Reply:
I can make a similar ClassModel and Controller which will handle the exceptions . This will leave only app.UseExceptionHandler("/error") in program.cs . Will that be a good approach ?

Reply:
Not sure that I understand how Controller will handle exceptions. Could you please elaborate on that?

Reply:
https://pastebin.com/wyZ1ZjVb  In this Pastebin is from where the idea of using controller comes . I think of injecting the dependencies in the constructor .The path in app.UseExceptionHandler("/error") and in controller [Route("error")] are the same. This will  allow the controller to handle exceptions redirected from exception handler middleware
 The other ways i have found use more than one class and a lot of try catch blocks .

Reply:
Ok. Let's try such an approach.